boards/arm/beagle_play_cc1352/doc/index.rst

```diff
@@ -0,0 +1,179 @@
++-------+--------------+-------------------------------------+
+| DIO16 | CC1352_TDO   | TAG-CONNECT TDO                     |
++-------+--------------+-------------------------------------+
+| DIO17 | CC1352_TDI   | TAG-CONNECT TDI                     |
++-------+--------------+-------------------------------------+
+| DIO18 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO19 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO20 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO21 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO22 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO23 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO24 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO25 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO26 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO27 | LED1         | CC1352_LED1 yellow LED9             |
++-------+--------------+-------------------------------------+
+| DIO28 | LED2         | CC1352_LED2 yellow LED8             |
++-------+--------------+-------------------------------------+
+| DIO29 | RF_PA        | Antenna mux PA enable               |
++-------+--------------+-------------------------------------+
+| DIO30 | RF_SUB1G     | Antenna mux SubG enable             |
++-------+--------------+-------------------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To flash, disable the existing driver that ties up the serial port and use
+the customized BSL Python script.
+
+1. Ensure the bcfserial driver isn’t blocking the serial port.
+
+   .. code-block:: console
+
+      echo "    fdtoverlays /overlays/k3-am625-beagleplay-bcfserial-no-firmware.dtbo" | sudo tee -a /boot/firmware/extlinux/extlinux.conf
+      sudo shutdown -r now
+
+2. Install CC1352-flasher
+
+   .. code-block:: console
+
+      sudo apt install python3-pip
+      pip3 install cc1352-flasher
+
+3. Flash the CC1352P7
+
+   .. code-block:: console
+
+      west flash
+
+Debugging
+=========
+
+For debugging, you can use the serial port or JTAG. You can use OpenOCD
+over the Tag-Connect header on the board.
+
+* https://docs.beagleboard.org/latest/accessories/cables.html#tagconnect-jtag
+
+References
+**********
+
+BeagleBoard.org BeaglePlay reference:
+  https://beagleplay.org
```

Comment:
please check how to make these proper "references" , see other boards like ex. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/bl5340_dvk/doc/index.rst?plain=1